### PR TITLE
Correctly install the app on all device just using app name + helper functions

### PR DESCRIPTION
**Pull request by CremaFR**

### CODE REVIEW

**Comment by CremaFR** [comment]
TODO. 

remove lib folder and use npm module instead when ready.

File: src/App.js [hunk 1]
```diff
@@ -1,38 +1,58 @@
...
-
+import { getDeviceInfo, installApp, getAppsListByDevice, openApp, quitCurrentApp, getCurrentAppAndVersion } from './lib';
 function App() {
   let transport;
   const [error, setError] = useState("");
   const onClickInstall = async (appName, isDelete) => {
     try {
       transport = await TransportWebUSB.create();
+      const deviceInfo = await getDeviceInfo(transport);
+      console.log(deviceInfo);
```
Comment (henri-ly) [inline src/App.js]: console.log()

File: src/App.js [hunk 2]
```diff
@@ -1,38 +1,58 @@
...
-        (a) =>
-          a.name.toLowerCase() === appName.toLowerCase() && a.category !== 2
-      );
-
-      // TODO get device version
-      const app = family.application_versions.find(
-        (a) => a.firmware === "nanox/2.0.2-2/ATOM/app_2.34.6"
-      );
+      const myApp = appByDevice.filter( app => app.name == appName);
+      console.log(myApp);
```
Comment (henri-ly) [inline src/App.js]: same

File: src/App.js [hunk 3]
```diff
@@ -1,38 +1,58 @@
...
+      await quitCurrentApp(transport);
+    } catch (e) {
+      setError(String(e));
+    }
+  }; 
+  const onClickGetCurrentAppAndVersion = async () => {
+    try {
+      transport = await TransportWebUSB.create();
+      const info = await getCurrentAppAndVersion(transport);
+      console.log(info);
```
Comment (henri-ly) [inline src/App.js:32]: console.log() here also

**Review by henri-ly — APPROVED** [review]
LGTM otherwise

**Pull request merged**